app/ui/lib/Pagination.tsx

```diff
@@ -50,7 +48,6 @@ export const Pagination = ({
     <>
       <div
         className={cn(
-          type === 'page' && 'py-5',
```

Comment:
This is the weird bit. In all the calling code above this, we were always `page`, never `inline`, but because this defaults to `inline`, we take this bit out.